### DISCUSSION TITLE
How does current version of backend settings sync work?

### DISCUSSION THREAD

**Discussion by matthewdias** [VueTorrent/VueTorrent#2103] [Q&A] [answered]
After updating to 2.19.0, the backend url settings field is gone. How does one configure the backend connection now?

**Comment by Larsluph** [comment, marked as answer]
You must upgrade the backend to use 2.0+
You can find upgrade steps in the [backend wiki](https://github.com/VueTorrent/vuetorrent-backend/wiki/Migration-Guide)
